src/cljs/nr/utils.cljs

```diff
@@ -307,9 +302,19 @@
 (defn render-message
   "Render icons, cards and special codes in a message"
   [input]
-  (if (string? input)
```

Comment:
simplified all these `render-` functions to just wrap `render-input` which can take a string or a vector of elements, so this guard isn't necessary anymore (I think it originally was for `render-cards`)